client/dashboard/sites/deployments/get-deployment-time-tooltip-text.tsx

```diff
@@ -0,0 +1,41 @@
+import { DeploymentRunStatus } from '@automattic/api-core';
```

Comment:
Should this be:
```suggestion
import type { DeploymentRunStatus } from '@automattic/api-core';
```

Reply:
Thanks for suggesting it. Done, https://github.com/Automattic/wp-calypso/pull/105905/commits/168a90673b93c3f10a351059f2c200a908b5f44c